app/experimenter/experiments/api/v5/__init__.py

```diff
@@ -1,6 +1,6 @@
 import graphene
 
-from experimenter.experiments.api.v5.mutation import Mutation
-from experimenter.experiments.api.v5.query import Query
+from experimenter.experiments.api.v5.mutations import Mutation
+from experimenter.experiments.api.v5.queries import Query
```

Comment:
Do these need renaming? I guess they don't *not* need renaming.

Reply:
Well it was more about consistency since a lot of the other files are pluralized so everytime i was typing out test paths it was inconsistent